src/api/RebalancingAPI.ts

```diff
@@ -184,6 +184,20 @@ export class RebalancingAPI {
     return await this.rebalancingSetToken.settleRebalance(rebalancingSetTokenAddress, txOpts);
   }
 
+  /**
+   * Ends failed auction and either returns to Default if no bids or sets to Drawdown if there are bids
+   *
+   * @param  rebalancingSetTokenAddress     Address of the Rebalancing Set
+   * @param  txOpts                         Transaction options object conforming to `Tx` with signer, gas, and
+   *                                          gasPrice data
+   * @return                                Transaction hash
+   */
+  public async endFailedAuctionAsync(rebalancingSetTokenAddress: Address, txOpts: Tx): Promise<string> {
```

Comment:
This doesn't have to match the name of the wrapper method. In fact, i think we could probably improve naming on this layer.